requirements.txt

```diff
@@ -1,6 +1,7 @@
 Flask-APScheduler==1.12.2
 Flask-Caching==1.10.1
 Flask==2.0.1
+werkzeug==2.0.3
```

Comment:
Why is this needed now?

Reply:
will have to check by removing it. It might just be an artifact of taking changes from integration branch where I was testing it.

Reply:
removed it